src/drivers/test/diff/ExaminedValue.sarif

```diff
@@ -1,21 +1,27 @@
-{
-    "all": {
-        "+": 0,
-        "-": 0
-    },
-    "error": {
-        "+": 0,
-        "-": 0,
-        "codes": []
-    },
-    "warning": {
-        "+": 0,
-        "-": 0,
-        "codes": []
-    },
-    "note": {
-        "+": 0,
-        "-": 0,
-        "codes": []
-    }
+{
+    "all": {
+        "+": 0,
+        "-": 1
+    },
+    "error": {
+        "+": 0,
+        "-": 0,
+        "codes": []
+    },
+    "warning": {
+        "+": 0,
+        "-": 1,
+        "codes": [
+            [
+                "cpp/drivers/examined-value Result of call to func2 is ignored; 77% of calls to this function have their result checked. Checked return values = 7 total calls = 9",

```

Comment:
It seems like there are a lot of diff files that got mistakenly checked in.  When I run the tests on this branch they come back clean.  Please re-run them and update this PR.